core/src/main/java/com/orange/ouds/core/theme/OudsColorScheme.kt

```diff
@@ -0,0 +1,1121 @@
+/*
+ * Software Name: OUDS Android
+ * SPDX-FileCopyrightText: Copyright (c) Orange SA
+ * SPDX-License-Identifier: MIT
+ *
+ * This software is distributed under the MIT license,
+ * the text of which is available at https://opensource.org/license/MIT/
+ * or see the "LICENSE" file for more details.
+ *
+ * Software description: Android library of reusable graphical components
+ */
+
+package com.orange.ouds.core.theme
+
+import androidx.compose.material3.ColorScheme
+import androidx.compose.material3.darkColorScheme
+import androidx.compose.material3.lightColorScheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.runtime.Stable
+import androidx.compose.ui.graphics.Color
+import com.orange.ouds.theme.tokens.OudsColorKeyToken
+import com.orange.ouds.theme.tokens.semantic.OudsColorSemanticTokens
+
+data class OudsColorScheme(
+    val globalColors: Global,
+    val actionColors: Action,
+    val alwaysColors: Always,
+    val backgroundColors: Background,
+    val borderColors: Border,
+    val brandColors: Brand,
+    val contentColors: Content,
+    val elevationColors: Elevation,
+    val gradientColors: Gradient,
+    val decorativeColors: Decorative,
+) {
+
+    data class Global(
+        val transparentDefault: Color
+    )
+
+    data class Action(
```

Comment:
Should we split negative, primary and secondary parameters into specific objects?

Same question for all nested classes.